generator/internal/connection_generator.cc

```diff
@@ -247,6 +249,7 @@ class $connection_class_name$ {
 std::shared_ptr<$connection_class_name$> Make$connection_class_name$(
 )""");
   HeaderPrint("    ");
+  if (IsExperimental()) HeaderPrint("ExperimentalTag, ");
```

Comment:
Consider refactoring the code to generate the `Make$connection_class_name$` arguments to some helper functions.  So maybe this will just be:

```cc
  HeaderPrint("    " + ConnectionFactoryFunctionArguments());
```

just a thought.
This code is getting complex enough that maybe we should refactor it to its

Reply:
Done, but I don't love it.